automation/test.sh

```diff
@@ -33,11 +33,16 @@ export WORKSPACE="${WORKSPACE:-$PWD}"
 if [[ $TARGET =~ openshift-.* ]]; then
   if [[ $TARGET =~ .*-crio-.* ]]; then
     export KUBEVIRT_PROVIDER="os-3.10.0-crio"
+  elif [[ $TARGET =~ .*-multus-.* ]]; then
+    export KUBEVIRT_PROVIDER="os-3.10.0-multus"
+    docker pull docker.io/kubevirtci/os-3.10.0-multus@sha256:d9a8df2ba545e579c933d96d33406a1a6b0dca37a81469fbc08bbb1d557d02f3
```

Comment:
Is this line here intentional?

Reply:
this is for debug, I am trying to find a problem with the openshift-multus doesn't run on the STD-CI. I will remove this after I find the problem